.github/workflows/main.yml

```
@@ -130,5 +130,44 @@ jobs:
         with:
          name: app reports
          path: app/build/test-results
-      
+
+  bazel_build_app:
+    name: Build Binary with Bazel
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup bazel
+        uses: jwlawson/actions-setup-bazel@v1
+        with:
+          bazel-version: '3.4.1'
+      - name: Clone Oppia Bazel
+        run: git clone https://github.com/miaboloix/bazel.git $HOME/oppia-bazel
+      - name: Set up JDK 9
+        uses: actions/setup-java@v1
+        with:
+          java-version: 9
+      - name: Create tmp/android_tools
+        run: |
+          mkdir -p $GITHUB_WORKSPACE/tmp/android_tools
+      - name: Extract Android Tools
+        run: |
```

```suggestion
      - name: Extract Android Tools
        run: |
          mkdir -p $GITHUB_WORKSPACE/tmp/android_tools
```

Probably can collapse these two for simplicity.